### PR TITLE
add angular course free from Fernando Herrera

### PR DESCRIPTION
**Pull request by dave-ramirez**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-es.md [hunk 1]
```diff
@@ -186,6 +186,7 @@
 
 ### Programación Web & Móvil
 
+* [Angular: Convierte cualquier template HTML en una WebAPP - Gratis](udemy.com/course/html-hacia-angular/)
```
Comment (eshellman) [inline courses/free-courses-es.md:189]: malformed link

**Comment by dave-ramirez** [comment]
adjustment done!

File: courses/free-courses-es.md (hunk 1 above)
Comment (dave-ramirez) [reply courses/free-courses-es.md]: adjustment done!

File: courses/free-courses-es.md [hunk 2]
```diff
@@ -186,6 +186,7 @@
 
 ### Programación Web & Móvil
 
+* [Angular: Convierte cualquier template HTML en una WebAPP - Gratis](https://www.udemy.com/course/html-hacia-angular/)
```
Comment (eshellman) [inline courses/free-courses-es.md:189]:
```suggestion
* [Angular: Convierte cualquier template HTML en una WebAPP - Gratis](https://www.udemy.com/course/html-hacia-angular/) - Fernando Herrera (Udemy)

**Pull request merged**